### PR TITLE
test3

### PR DESCRIPTION
**Pull request by planteater**
afasfdasdf

**This PR Addresses:**  
[Another horrible bug!](https://trello.com/c/icPu2wJW/492-another-horrible-bug)

### CODE REVIEW

File: tracking.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	constructor() {
+		this.disposable = window.registerUriHandler(this);
+	}
+
+	async beerHandleUri(uri: Uri) {
+		await Container.webview.handleProtocol(uri);
+	}
+
+	dispose(): fillthevoid {
+		this.disposable && this.disposable.dispose();
```
Comment (planteater) [inline tracking.js:17]: Appear in real time?

Comment (codestreamtest) [reply tracking.js:17]: All the time

Comment (planteater) [reply tracking.js:17]: Pops in?

Comment (planteater) [reply tracking.js:17]: Nice

File: tracking.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+"use strict";
+import { Disposable, Uri, UriHandler, window } from "vscode";
+import { Container } from "./container";
+
+export class ProtocolHandler implements UriHandler {
+	private disposable: Disposable;
+
+	constructor() {
+		this.disposable = window.registerUriHandler(this);
```
Comment (planteater) [inline tracking.js:9]: Pending pending pending.

File: tracking.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+"use strict";
+import { Disposable, Uri, UriHandler, window } from "vscode";
+import { Container } from "./container";
```
Comment (planteater) [inline tracking.js:3]: pending 2

File: tracking.js (hunk 2 above)
Comment (planteater) [inline tracking.js:9]: pending 3

Comment (planteater) [reply tracking.js:9]: reply to "pending 3"

File: tracking.js (hunk 3 above)
Comment (planteater) [reply tracking.js:3]: reply to p2

Comment (planteater) [reply tracking.js:3]: again to p2

File: tracking.js (hunk 2 above)
Comment (planteater) [reply tracking.js:9]: reply to p1

**Pull request closed without merging**